src/riskassessment/riskassessmentinterfaces.ts

```diff
@@ -10,6 +11,8 @@ export enum RiskAssessmentCycle {
   BiYearly = 5,
 }
 
+export const RiskAssessmentCycleSchema = Joi.number().max(5).min(1);
```

Comment:
I think it would be good to have a test for this as it will break runtime if someone forgets to adjust these values here.

So I think it would be good to have a simple test like this for it:
```
for (let item in RiskAssessmentCycle) {
    if (Number.isInteger(Number(item))) {
        RiskAssessmentCycleSchema.validate(item);
    }
}
```